docs/build/guides/building-a-voting-dapp.md

```diff
@@ -768,14 +788,19 @@ async function listModules() {
   console.log(response.result.data);
 }
 ```
-
 :::
 
-You can use the snippets below to deploy your contract to **chain 0** on `testnet` and `mainnet`:
+The snippets can also be found in the [tutorial repo](https://github.com/kadena-community/kadena.js/tree/master/packages/tutorials/election-dapp).
+
+You can use the snippet below to deploy your contract to **chain 0** on `testnet`. To do this, it's required to run Chainweaver
```

Comment:
we should probably encourage users to deploy on a different chain than chain 0 (probably in the 10+ range)

Reply:
For now I suggest we use `1` everywhere in this tutorial for users to have a smooth ride, until we have such options made available for them.